server.js

```diff
@@ -72,9 +71,40 @@ app.use(session({
     saveUninitialized: false
 }));
 
-app.listen(port, function() {
+/*
+    HTTPS protocol
+*/
+
+// Change to 443 in production
+const httpsPort = 8080;
+// Change to port 80 in production
+const httpPort = 8000;
+// Change to production website name
+const hostName = '127.0.0.1';
+
+var forceSSL = require('express-force-ssl');
+var http = require('http');
+var https = require('https');
+
+var ssl_options = {
+  key: fs.readFileSync('./keys/private.key'),
+  cert: fs.readFileSync('./keys/cert.crt')
+};
+
+var secureServer = https.createServer(ssl_options, app);
+
+var httpServer = http.createServer(function(req, res) {
+    // Redirects to https location
+    res.writeHead(301, {"Location": "https://" + hostName + ':' + httpsPort});
+    res.end();
+});
+
+app.use(forceSSL);
+
+//HTTPS server
+secureServer.listen(httpsPort,function(){
     logger.log('//------------------------');
-    logger.log(common.formatString('Server listening on http://localhost:{0}.', [port]));
+    logger.log(common.formatString('Secure Server listening on https://'+hostName+':{0}.', [httpsPort]));
```

Comment:
this can be written as:
`logger.log(common.formatString('Secure Server listening on https://{0}:{1}.', [hostName, httpsPort]));`
:)

Reply:
Or even using string interpolation (I was planning on going to change all string formatting using this)
```javascript
logger.log(`Secure Server listening on https://${hostname}:${httpsPort}.`);